content/docs/command-reference/import-url.md

```diff
@@ -136,9 +136,13 @@ produces a regular stage in `dvc.yaml`.
 
 - `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
   that the data source is valid). This is useful if you need to define the
-  project imports quickly, and download everything later (use `dvc update` to
-  finish the operation(s)); or if the target data already exist locally and you
-  want to "DVCfy" this state of the project (see also `dvc commit`).
+  project imports quickly, and import the data later (use `dvc update` to finish
+  the operation(s)).
+
+- `--no-download` - create the import `.dvc` with data checksums but without
+  downloading the associated data. This is useful if you need track changes in
+  remote data but do not (yet) need to download data to the local workspace.
+  Data can be later downloaded using `dvc pull`.
```

Comment:
When pulling `.dvc` files created with `--no-download` (partial imports), the deps checksum will be verified and `pull` will error out if these are changed as there's currently no way of retrieving previous version (although this is being worked on in https://github.com/iterative/dvc/pull/8164.

Not sure if this is worth mention here, or whether we should modify the pull command docs, mentioning the special behaviour of pull with partial imports

Reply:
Maybe just add a note at the end of here like "but will fail if the data matching the checksum is no longer available."

Reply:
Done